### PR TITLE
add css shapes and grid

### PR DESCRIPTION
**Pull request by potch**

### CODE REVIEW

File: features/css-shapes.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: CSS Shapes
+category: css, layout
+bugzilla: 1040714
+firefox_status: in-development
+mdn_url:
```
Comment (marco-c) [inline features/css-shapes.md]: Can you add MDN URLs for both features?

File: features/css-shapes.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+spec_url: http://www.w3.org/TR/css-shapes-1/
+spec_repo:
```
Comment (marco-c) [inline features/css-shapes.md]: Can you add the spec repo for both features? (or remove the `spec_repo` field altogether)

**Comment by potch** [comment]
Is this good to merge?

File: features/css-shapes.md (hunk 1 above)
Comment (potch) [inline features/css-shapes.md]: couldn't really find good MDN docs for these features, can look some more.

File: features/css-shapes.md (hunk 2 above)
Comment (potch) [inline features/css-shapes.md]: couldn't find a spec repo for these features- will remove

File: features/css-shapes.md (hunk 1 above)
Comment (digitarald) [inline features/css-shapes.md]: Found a rather empty page: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Shapes

**Comment by marco-c** [comment]
@potch ping

**Comment by potch** [comment]
Made requested changes.

**Pull request merged**